cranelift/module/src/module.rs

```diff
@@ -307,6 +313,12 @@ impl std::convert::From<CodegenError> for ModuleError {
     }
 }
 
+impl std::convert::From<SetError> for ModuleError {
+    fn from(source: SetError) -> Self {
+        Self::Flag { 0: source }
```

Comment:
I didn't know this was legal syntax for constructing tuple-like structs and enum variants! I guess it's the prevailing style here but I'm not sure it should be. :sweat_smile:

Reply:
It's not how I would normally do it, but in this case I copied the other impl just above.